src/Intersection.jl

```diff
@@ -514,7 +514,8 @@ end
 function constraints_list(cap::Intersection{N, S1, S2}) where {N<:Real,
                           S1<:AbstractPolytope{N}, S2<:AbstractPolytope{N}}
     constraints = [constraints_list(cap.X); constraints_list(cap.Y)]
-    return remove_redundant_constraints!(constraints)
+    remove_redundant_constraints!(constraints)
```

Comment:
good catch. then shouldn't we do:

```julia
if remove_redundant_constraints!(constraints)
    return constraints
else
    return EmptySet{N}()
end
```

Reply:
No, `constraints_list` does not return a set.